README.md

```diff
@@ -91,19 +91,14 @@ Uses stories from [Storybook](https://storybook.js.org/) as test cases. In other
 ### Setup
```

Comment:
Maybe we should describe how to install required pnpm version. I'm not sure we could easily migrate to next major version.
Ex. `npm i -g pnpm@8.3.1`